Capstone-next related OpenBSD/sparc64 build failure

<!-- This template is meant for bug reports, if you have a feature request, please be as descriptive as possible and delete the template

Make sure you are testing using the latest git version of rizin before submitting any issue.

If you would like to report a bug, please fill the template below
-->

### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | OpenBSD 7.4 sparc64
| `rizin -v` full output, **not truncated** (mandatory)  | d11a76349745576534fda87dd43d77d803a9bd86 (one commit over 6eeb9e4f8321a8095c91b6e847187e96223d9bfa)


### Actual behavior

```
[263/1953] Compiling C object subprojects/zstd-1.5.5/lib/libzstd.a.p/decompress_zstd_decompress_block.c.o
../subprojects/zstd-1.5.5/lib/decompress/zstd_decompress_block.c:1350: warning: ignoring #pragma GCC optimize
../subprojects/zstd-1.5.5/lib/decompress/zstd_decompress_block.c:1567: warning: ignoring #pragma GCC optimize
[320/1953] Compiling C object librz/util/librz_util.so.0.7.0.p/sys.c.o
../librz/util/sys.c: In function 'rz_sys_perror_str':
../librz/util/sys.c:772: warning: ignoring #pragma push_macro
../librz/util/sys.c:775: warning: ignoring #pragma pop_macro
[329/1953] Compiling C object librz/util/librz_util.so.0.7.0.p/thread.c.o
../librz/util/thread.c: In function 'rz_th_set_affinity':
../librz/util/thread.c:185: warning: ignoring #pragma message
[391/1953] Linking target librz/util/sdb_native
librz/util/librz_util.so.0.7.0: warning: strcat() is almost always misused, please use strlcat()
librz/util/librz_util.so.0.7.0: warning: sprintf() is often misused, please use snprintf()
librz/util/librz_util.so.0.7.0: warning: strcpy() is almost always misused, please use strlcpy()
librz/util/librz_util.so.0.7.0: warning: rand() may return deterministic values, is that what you want?
[682/1953] Compiling C object subprojects/tree-sitter-c/libtree-sitter-c.a.p/src_parser.c.o
../subprojects/tree-sitter-c/src/parser.c:4: warning: expected [error|warning|ignored] after '#pragma GCC diagnostic'
[751/1953] Compiling C object librz/bin/librz_bin.so.0.7.0.p/format_elf_elf_info.c.o
../librz/bin/format/elf/elf_info.c: In function 'file_type_is_processor_specific':
../librz/bin/format/elf/elf_info.c:811: warning: comparison is always true due to limited range of data type
[757/1953] Compiling C object librz/bin/librz_bin.so.0.7.0.p/format_elf_elf64_info.c.o
In file included from ../librz/bin/format/elf/elf64_info.c:6:
../librz/bin/format/elf/elf_info.c: In function 'file_type_is_processor_specific':
../librz/bin/format/elf/elf_info.c:811: warning: comparison is always true due to limited range of data type
[771/1953] Compiling C object librz/bin/librz_bin.so.0.7.0.p/format_java_class_bin.c.o
../librz/bin/format/java/class_bin.c: In function 'rz_bin_java_class_version':
../librz/bin/format/java/class_bin.c:319: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:321: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:323: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:325: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:327: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:329: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:331: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:333: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:335: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:337: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:339: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:341: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:343: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:345: warning: comparison is always true due to limited range of data type
../librz/bin/format/java/class_bin.c:347: warning: comparison is always true due to limited range of data type
[1276/1953] Linking target librz/asm/librz_asm.so.0.7.0
FAILED: librz/asm/librz_asm.so.0.7.0
cc  -o librz/asm/librz_asm.so.0.7.0 librz/asm/librz_asm.so.0.7.0.p/asm.c.o librz/asm/librz_asm.so.0.7.0.p/aop.c.o librz/asm/librz_asm.so.0.7.0.p/acode.c.o librz/asm/librz_asm.so.0.7.0.p/binutils_as.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_6502.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_8051.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_amd29k.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_arm_as.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_arm_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_avr.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_bf.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_chip8.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_cil.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_cr16.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_dalvik.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_dcpu16.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_ebc.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_gb.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_h8300.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_hexagon.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_i4004.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_i8080.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_java.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_lh5801.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_lm32.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_luac.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_m68k_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_m680x_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_malbolge.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_mcs96.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_mips_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_msp430.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_null.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_or1k.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_pic.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_mcore.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_ppc_as.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_ppc_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_propeller.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_pyc.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_rl78.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_rsp.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_sh.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_snes.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_sparc_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_spc700.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_sysz.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_tms320.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_tms320c64x.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_v810.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_v850.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_wasm.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_x86_as.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_x86_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_x86_nasm.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_x86_nz.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_xap.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_xcore_cs.c.o librz/asm/librz_asm.so.0.7.0.p/arch_amd29k_amd29k.c.o librz/asm/librz_asm.so.0.7.0.p/arch_8051_8051_ass.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arm_armass.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arm_armass64.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arm_arm_it.c.o librz/asm/librz_asm.so.0.7.0.p/arch_avr_disassembler.c.o librz/asm/librz_asm.so.0.7.0.p/arch_avr_assembler.c.o librz/asm/librz_asm.so.0.7.0.p/arch_cil_cil_dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_cr16_cr16_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_ebc_ebc_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_h8300_h8300_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_hexagon_hexagon.c.o librz/asm/librz_asm.so.0.7.0.p/arch_hexagon_hexagon_arch.c.o librz/asm/librz_asm.so.0.7.0.p/arch_hexagon_hexagon_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_java_jvm.c.o librz/asm/librz_asm.so.0.7.0.p/arch_java_assembler.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_lua_arch.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_v54_opcode_54.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_v54_disassembly_54.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_v54_assembly_54.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_v53_opcode_53.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_v53_disassembly_53.c.o librz/asm/librz_asm.so.0.7.0.p/arch_luac_v53_assembly_53.c.o librz/asm/librz_asm.so.0.7.0.p/arch_mips_mipsasm.c.o librz/asm/librz_asm.so.0.7.0.p/arch_msp430_msp430_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_or1k_or1k_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pic_pic_baseline.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pic_pic_midrange.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pic_pic_pic18.c.o librz/asm/librz_asm.so.0.7.0.p/arch_mcore_mcore.c.o librz/asm/librz_asm.so.0.7.0.p/arch_ppc_libvle_vle.c.o librz/asm/librz_asm.so.0.7.0.p/arch_ppc_libps_libps.c.o librz/asm/librz_asm.so.0.7.0.p/arch_propeller_propeller_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_10.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_11.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_12.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_13.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_14.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_15.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_16.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_20.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_21.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_22.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_23.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_24.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_25.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_26.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_27.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_2x.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_30.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_31.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_32.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_33.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_34.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_35.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_36.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_37.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_38.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_39.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_310.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_3x.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_analysis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode_arg_fmt.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_opcode.c.o librz/asm/librz_asm.so.0.7.0.p/arch_pyc_pyc_dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_rl78_rl78_instr.c.o librz/asm/librz_asm.so.0.7.0.p/arch_rl78_rl78_maps.c.o librz/asm/librz_asm.so.0.7.0.p/arch_rl78_rl78_operand.c.o librz/asm/librz_asm.so.0.7.0.p/arch_rl78_rl78.c.o librz/asm/librz_asm.so.0.7.0.p/arch_rsp_rsp_idec.c.o librz/asm/librz_asm.so.0.7.0.p/arch_sh_disassembler.c.o librz/asm/librz_asm.so.0.7.0.p/arch_sh_assembler.c.o librz/asm/librz_asm.so.0.7.0.p/arch_sh_lookup.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_c55x_plus_c55plus.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_c55x_plus_c55plus_decode.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_c55x_plus_decode_funcs.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_c55x_plus_hashtable.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_c55x_plus_ins.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_c55x_plus_utils.c.o librz/asm/librz_asm.so.0.7.0.p/arch_tms320_tms320_dasm.c.o librz/asm/librz_asm.so.0.7.0.p/arch_v810_v810_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_v850_v850_disas.c.o librz/asm/librz_asm.so.0.7.0.p/arch_wasm_wasm.c.o librz/asm/librz_asm.so.0.7.0.p/arch_z80_z80.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_riscv_cs.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_tricore_cs.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arc_gnu_arc-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arc_gnu_arc-ext.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arc_gnu_arc-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_arc_gnu_arcompact-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_cris_gnu_cris-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_cris_gnu_cris-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_hppa_gnu_hppa-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_lanai_gnu_lanai-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_lanai_gnu_lanai-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_mips_gnu_mips-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_mips_gnu_mips-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_mips_gnu_mips16-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_nios_gnu_nios2-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_nios_gnu_nios2-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_sparc_gnu_sparc-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_sparc_gnu_sparc-opc.c.o librz/asm/librz_asm.so.0.7.0.p/arch_vax_vax-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_xtensa_gnu_elf32-xtensa.c.o librz/asm/librz_asm.so.0.7.0.p/arch_xtensa_gnu_xtensa-dis.c.o librz/asm/librz_asm.so.0.7.0.p/arch_xtensa_gnu_xtensa-isa.c.o librz/asm/librz_asm.so.0.7.0.p/arch_xtensa_gnu_xtensa-modules.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_arc.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_cris_gnu.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_hppa_gnu.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_lanai_gnu.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_mips_gnu.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_nios2.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_riscv.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_sparc_gnu.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_vax.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_xtensa.c.o librz/asm/librz_asm.so.0.7.0.p/p_asm_z80.c.o -Wl,--as-needed -Wl,--allow-shlib-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,librz_asm.so.0.7 -Wl,--whole-archive subprojects/rzspp/librzspp.a -Wl,--no-whole-archive -Wl,-z,origin '-Wl,-rpath,$ORIGIN/../util:$ORIGIN/../config:$ORIGIN/../syscall:$ORIGIN/../flag:$ORIGIN/../parse:$ORIGIN/../reg:$ORIGIN/../cons:$ORIGIN/../bin:$ORIGIN/../demangler:$ORIGIN/../socket:$ORIGIN/../magic:$ORIGIN/../hash:$ORIGIN/../io:$ORIGIN/../crypto:$ORIGIN/../type' -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/util -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/config -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/syscall -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/flag -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/parse -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/reg -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/cons -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/bin -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/demangler -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/socket -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/magic -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/hash -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/io -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/crypto -Wl,-rpath-link,/home/florian/dev/rizin/build/librz/type librz/util/librz_util.so.0.7.0 librz/config/librz_config.so.0.7.0 librz/syscall/librz_syscall.so.0.7.0 librz/flag/librz_flag.so.0.7.0 librz/parse/librz_parse.so.0.7.0 librz/bin/librz_bin.so.0.7.0 subprojects/capstone-next/libcapstone.a -Wl,--end-group
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isValid':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:326: multiple definition of `isValid'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:326: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isFloatingPoint':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:333: multiple definition of `isFloatingPoint'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:333: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isInteger':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:344: multiple definition of `isInteger'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:344: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isScalarInteger':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:359: multiple definition of `isScalarInteger'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:359: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:366: multiple definition of `isVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:366: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isScalableVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:374: multiple definition of `isScalableVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:374: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `isFixedLengthVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:380: multiple definition of `isFixedLengthVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:380: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is16BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:387: multiple definition of `is16BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:387: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is32BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:396: multiple definition of `is32BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:396: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is64BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:408: multiple definition of `is64BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:408: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is128BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:422: multiple definition of `is128BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:422: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is256BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:437: multiple definition of `is256BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:437: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is512BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:453: multiple definition of `is512BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:453: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is1024BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:469: multiple definition of `is1024BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:469: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `is2048BitVector':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:483: multiple definition of `is2048BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:483: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `getVectorElementType':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:496: multiple definition of `getVectorElementType'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:496: first defined here
subprojects/capstone-next/libcapstone.a(arch_ARM_ARMMapping.c.o): In function `getVectorMinNumElements':
../subprojects/capstone-next/arch/ARM/../../cs_simple_types.h:688: multiple definition of `getVectorMinNumElements'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:688: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isValid':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:326: multiple definition of `isValid'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:326: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isFloatingPoint':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:333: multiple definition of `isFloatingPoint'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:333: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isInteger':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:344: multiple definition of `isInteger'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:344: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isScalarInteger':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:359: multiple definition of `isScalarInteger'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:359: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:366: multiple definition of `isVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:366: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isScalableVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:374: multiple definition of `isScalableVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:374: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `isFixedLengthVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:380: multiple definition of `isFixedLengthVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:380: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is16BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:387: multiple definition of `is16BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:387: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is32BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:396: multiple definition of `is32BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:396: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is64BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:408: multiple definition of `is64BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:408: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is128BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:422: multiple definition of `is128BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:422: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is256BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:437: multiple definition of `is256BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:437: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is512BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:453: multiple definition of `is512BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:453: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is1024BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:469: multiple definition of `is1024BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:469: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `is2048BitVector':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:483: multiple definition of `is2048BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:483: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `getVectorElementType':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:496: multiple definition of `getVectorElementType'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:496: first defined here
subprojects/capstone-next/libcapstone.a(arch_PowerPC_PPCMapping.c.o): In function `getVectorMinNumElements':
../subprojects/capstone-next/arch/PowerPC/../../cs_simple_types.h:688: multiple definition of `getVectorMinNumElements'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:688: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isValid':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:326: multiple definition of `isValid'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:326: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isFloatingPoint':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:333: multiple definition of `isFloatingPoint'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:333: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isInteger':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:344: multiple definition of `isInteger'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:344: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isScalarInteger':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:359: multiple definition of `isScalarInteger'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:359: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:366: multiple definition of `isVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:366: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isScalableVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:374: multiple definition of `isScalableVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:374: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `isFixedLengthVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:380: multiple definition of `isFixedLengthVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:380: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is16BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:387: multiple definition of `is16BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:387: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is32BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:396: multiple definition of `is32BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:396: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is64BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:408: multiple definition of `is64BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:408: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is128BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:422: multiple definition of `is128BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:422: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is256BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:437: multiple definition of `is256BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:437: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is512BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:453: multiple definition of `is512BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:453: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is1024BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:469: multiple definition of `is1024BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:469: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `is2048BitVector':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:483: multiple definition of `is2048BitVector'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:483: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `getVectorElementType':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:496: multiple definition of `getVectorElementType'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:496: first defined here
subprojects/capstone-next/libcapstone.a(arch_TriCore_TriCoreMapping.c.o): In function `getVectorMinNumElements':
../subprojects/capstone-next/arch/TriCore/../../cs_simple_types.h:688: multiple definition of `getVectorMinNumElements'
subprojects/capstone-next/libcapstone.a(arch_AArch64_AArch64Mapping.c.o):../subprojects/capstone-next/arch/AArch64/../../cs_simple_types.h:688: first defined here
collect2: ld returned 1 exit status
[1283/1953] Compiling C object librz/il/librz_il.so.0.7.0.p/il_export_string.c.o
ninja: build stopped: subcommand failed.
```

--- a/librz/analysis/arch/x86/il_fp_ops.inc
+++ b/librz/analysis/arch/x86/il_fp_ops.inc
@@ -627,6 +627,7 @@ IL_LIFTER(fcomi) {
 	return fcomi_helper(ins, pc, analysis);
 }
 
+#if CS_API_MAJOR > 4
 /**
  * FCOMIP
  * Compare ST(0) and ST(i), set the EFLAGS accordingly and pop the stack
@@ -634,6 +635,7 @@ IL_LIFTER(fcomi) {
 IL_LIFTER(fcomip) {
 	return SEQ2(fcomi_helper(ins, pc, analysis), x86_il_st_pop());
 }
+#endif
 
 /**
  * FCHS

--- a/librz/analysis/arch/x86/x86_il.c
+++ b/librz/analysis/arch/x86/x86_il.c
@@ -302,14 +302,16 @@ x86_il_ins x86_ins[X86_INS_ENDING] = {
 	[X86_INS_FCOMPP] = x86_il_fcompp,
 	[X86_INS_FICOMP] = x86_il_ficomp,
 	[X86_INS_FCOMI] = x86_il_fcomi,
+#if CS_API_MAJOR > 4
 	[X86_INS_FCOMPI] = x86_il_fcomip,
+	[X86_INS_FUCOMPI] = x86_il_fcomip,
+#endif
 	/* Using the same FCOM & FCOMI family IL lifters for FUCOM & FUCOMI family instructions
 	 * since we don't support invalid arithmetic operand exceptions (#IA) anyways. */
 	[X86_INS_FUCOM] = x86_il_fcom,
 	[X86_INS_FUCOMP] = x86_il_fcomp,
 	[X86_INS_FUCOMPP] = x86_il_fcompp,
 	[X86_INS_FUCOMI] = x86_il_fcomi,
-	[X86_INS_FUCOMPI] = x86_il_fcomip,
 	[X86_INS_FCHS] = x86_il_fchs,
 	[X86_INS_FTST] = x86_il_ftst,
 	[X86_INS_FRNDINT] = x86_il_frndint,

--- a/librz/analysis/p/analysis_arm_cs.c
+++ b/librz/analysis/p/analysis_arm_cs.c
@@ -1423,6 +1423,7 @@ jmp $$ + 4 + ( [delta] * 2 )
 			} else {
 				op->type = RZ_ANALYSIS_OP_TYPE_UJMP;
 			}
+			break;
 		}
 		if (ISIMM(1)) {
 			op->val = IMM(1);

--- a/subprojects/capstone-next.wrap
+++ b/subprojects/capstone-next.wrap
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/capstone-engine/capstone.git
-revision = cb2b87974d6bdc347578eddea4558802bb6a0c1b
+revision = 34a1e012b7eb8a3b03971cdf2d32b603855d5b09
 directory = capstone-next
 patch_directory = capstone-next
 depth = 1

--- a/test/db/analysis/arm
+++ b/test/db/analysis/arm
@@ -1282,3 +1282,38 @@ EXPECT=<<EOF
                `-----------'
 EOF
 RUN
+
+NAME=Check MOV gets identified as JUMP if it moves to PC
+FILE==
+ARGS=-a arm -b 16
+CMDS=<<EOF
+s 0x100
+wx bff36f8f2548874680bd000072b680bd
+s 0x19c
+wx 0x0c010000
+s 0x100
+aaa
+pd 10 @ 0x100
+px 4 @ 0x19c
+EOF
+EXPECT=<<EOF
+            ;-- pc:
+            ;-- r15:
+/ fcn.00000100();
+|           0x00000100      isb   sy
+|           0x00000104      ldr   r0, aav.0x0000010c                   ; [0x10c:4]=0xbd80b672 ; "r\xb6\x80\xbd"
+\           0x00000106      mov   pc, r0
+            0x00000108      pop   {r7, pc}
+            0x0000010a      movs  r0, r0
+            ; CODE XREFS from fcn.00000100 @ 0x104, 0x106
+            ; UNKNOWN XREF from aav.0x0000010c @ +0x90
+/ aav.0x0000010c();
+|           0x0000010c      cpsid i
+\           0x0000010e      pop   {r7, pc}
+            0x00000110      movs  r0, r0
+            0x00000112      movs  r0, r0
+            0x00000114      movs  r0, r0
+- offset -   0 1  2 3  4 5  6 7  8 9  A B  C D  E F  0123456789ABCDEF
+0x0000019c  0c01 0000                                ....
+EOF
+RUN

--- a/test/db/archos/darwin-arm64/dbg
+++ b/test/db/archos/darwin-arm64/dbg
@@ -92,16 +92,20 @@ fl@F:maps~hello~[1-]
 echo --
 dm*~hello~[0-2]
 EOF
-REGEXP_FILTER_OUT=([0-9]+ [a-zA-Z0-9_-]+\.[\.rwx_]*|[f-].+)
+REGEXP_FILTER_OUT=([0-9]+ [a-zA-Z0-9_-]+\.[\.rwx_]+|[f-].+\n)
 EXPECT=<<EOF
 16384 hello_macos_arm64.r_x
 16384 hello_macos_arm64.rw
 16384 hello_macos_arm64.rw.
 16384 hello_macos_arm64.r
 --
+
 f+ map.hello_macos_arm64.r_x 0x00004000
+
 f+ map.hello_macos_arm64.rw 0x00004000
+
 f+ map.hello_macos_arm64.rw 0x00004000
+
 f+ map.hello_macos_arm64.r 0x00004000
 EOF
 RUN

